config/registry_image_pruner/image_pruner/tests/requirements.txt

```diff
@@ -1,2 +0,0 @@
-pytest
-flexmock
```

Comment:
It was probably not necessary to rework tests, no? These don't get installed in the job, only on the developer's machine.

Reducing the tests changes would make it possible to see what tests were added in this commit, currently it's hard to review

Reply:
The rework covers all previous existing tests.

With the replacement, the existing tests have to be reworked by mocking calls to urllib properly and it looks no difference too much with the current implementation except using different mock framework and test runner. And for a Go project, I would like to keep it simple enough with builtin libraries for a single Python script.

Reply:
ok